assets/tests/components/map.test.tsx

```diff
@@ -14,13 +14,7 @@ import { Shape } from "../../src/schedule"
 import vehicleFactory from "../factories/vehicle"
 import userEvent from "@testing-library/user-event"
 
-jest.unmock("leaflet")
-jest.unmock("react-leaflet-control")
-
-jest.mock("../../src/laboratoryFeatures", () => ({
-  __esModule: true,
-  default: jest.fn(() => false),
-}))
+window.scrollTo = jest.fn()
```

Comment:
Is it possible to make this update temporarily and restore the old value once the tests finish running? I worry about weird interactions with other tests that might reference this same function.

Reply:
Updated to only mock once in the 1 test where mocking is needed